java/scripts/docker-testing-pgsql.sh

```diff
@@ -3,10 +3,44 @@
 set -e
 
 TARGET="test"
-if [ -n "$1" ]; then
-    TARGET=$1
+
+help() {
+  echo ""
+  echo "Script to run pgsql Java unit tests"
+  echo ""
+  echo "Syntax: "
+  echo ""
+  echo "${SCRIPT} -c /path/to/oscrc [-t ant-target]"
+  echo ""
+  echo "Where: "
+  echo "  -c  Path to the OSC credentials (usually ~/.osrc)"
```

Comment:
Same.

Reply:
completely dropped those options since now I mount the oscrc in /root (home of the user running the script). osc then takes care of locating the file